### ISSUE TITLE
Under Debian the window is semi-transparent, and WM sees it as only 1 pixel

### ISSUE THREAD

**Issue by bestouff** [three-rs/three#90]
When running on of the examples (with rustc 1.18.0), the windows are always kind of transparent (see attached screenshot). What's more strange is that the window manager seems to see it as only 1 pixel wide and high (look at the top-left corner of the window, you'll see the window borders are stuck there) :
![threewindowtrouble](https://user-images.githubusercontent.com/1163188/29768893-43498400-8be8-11e7-82c1-88272230e183.png)

**Comment by vitvakatu** [comment]
Hello, thanks for testing. It seems like glutin issue actually, can you test glutin examples?

**Comment by bestouff** [comment]
Indeed it seems it's glutin related (I'm running a standard GNOME on Wayland):
```
[xav@pcxav:~/git/glutin]$ cargo run --example window 
    Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
     Running `target/debug/examples/window`
Pixel format of the window's GL context: PixelFormat { hardware_accelerated: true, color_bits: 24, alpha_bits: 8, depth_bits: 24, stencil_bits: 8, stereoscopy: false, double_buffer: true, multisampling: None, srgb: false }
OpenGL version 3.3 (Core Profile) Mesa 13.0.6
WindowEvent { window_id: WindowId(Wayland(WindowId(140706616329072))), event: Resized(1, 1) }
WindowEvent { window_id: WindowId(Wayland(WindowId(140706616329072))), event: Focused(true) }
WindowEvent { window_id: WindowId(Wayland(WindowId(140706616329072))), event: Resized(1, 1) }
```
I'll open a glutin bug.

**Comment by bestouff** [comment]
I think it's already there: tomaka/glutin#889 (at least for the pixel-size problem, not the transparency problem).

**Comment by vitvakatu** [comment]
@bestouff there is another winit issue: https://github.com/tomaka/winit/issues/163

**Comment by bestouff** [comment]
Yes it's referenced in the first one. I guess I'll have to wait a bit before playing with three ...

**Comment by vitvakatu** [comment]
Sad to hear that :(
I'm leaving this issue open, ping me if you find a solution that requires changes inside `three`

**Comment by kvark** [comment]
@bestouff the obvious workaround would be to run it through Xorg instead of Wayland

> I guess I'll have to wait a bit before playing with three ...

Most of the projects use `winit`/`glutin` for windowing. I suppose you can try `ggez` for it's using SDL.

**Comment by bestouff** [comment]
@kvark Yep I have rebooted (sometimes you have to:)) just to try under Xorg and it works fine.
